textx/scoping/providers.py

```diff
@@ -439,6 +441,10 @@ class GlobalRepo(ImportURI):
       * create FQNGlobalRepo
       * register models used for lookup into the scope provider
     Then the scope provider is ready to be registered and used.
+
+    The model parameter `project_root` (see _tx_model_params) can be used to
+    set a project directory, where all file patterns not referring to an
+    absolute file position are looked up.
```

Comment:
This explanation looks good, so might go to the docs for explanation of the parameter.

Reply:
Done. Retsricted to globalrepo. Also added a link to scoping.md (for the globalrepo)